services/bedrock/pom.xml

```diff
@@ -56,5 +56,10 @@
             <artifactId>aws-json-protocol</artifactId>
             <version>${awsjavasdk.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Will new services get this dependency from this point on?

Reply:
yes, added to the feature branch in https://github.com/aws/aws-sdk-java-v2/pull/4309